Snake.h

```diff
@@ -0,0 +1,76 @@
+#include <stdio.h>
+
+class Game;
```

Comment:
why cant you include the header file?

Reply:
which one, Game.h? it is not showing any error so I leave that as it is. will check up if you want me too include.